src/biosHandler.cpp

```diff
@@ -0,0 +1,57 @@
+#include "biosHandler.hpp"
+
```

Comment:
why there is a blank line here?

Reply:
Clang.

Reply:
then why there is no blank line in manager.hpp?

Reply:
I did not get the question. what is the relation to manager.hpp? 
I have no control on this. Clang is inserting a blank line there.

Reply:
Where as if you check other Cpp files it has that blank line.